### DISCUSSION TITLE
notepad++ vs virtual studio code

### DISCUSSION THREAD

**Discussion by bbannon** [craigk5n/webcalendar#309] [General]
I've been using notepad++ since it came out.
I've just discovered vscode from Microsoft.

It seems to be a combination of code editor, linter, formatter, git bash, powershell and gulp.  And other things. 
There are hundreds of extensions covering every computer language I know of; CSS, HTML, Javascript, PHP, composer, docker, SCSS, Python, Java and too many more to mention. 
As well as several people languages.
Best of all, there are shared configuration settings so the whole team is on the same page as far as formatting and such.
But, each user can still set personal preferences, such as screen color and font size, etc.

Is anyone else using it? 
What are your likes/dislikes?

**Comment by grzchr15** [comment]
biggest advantage is consistent code formating with correct settings

**Comment by Tharrington86** [comment]
I have an Angular webapp and use vscode for the majority of my frontend work because its typescript support is second to none in my opinion. Although, if I have a very quick change to make, I will still often open with notepad++ because of how lightweight it is.

**Comment by bbannon** [comment]
Interesting. I deleted notepad++ and just use vs code
Though I did install the notepad++ extension to convert shortcut keys over.
